fiftyone/plugins/core.py

```diff
@@ -278,15 +296,21 @@ def download_plugin(url_or_gh_repo, plugin_names=None, overwrite=False):
                 logger.debug(f"Skipping unwanted plugin '{plugin.name}'")
                 continue
 
-            existing_dir = existing_plugin_dirs.get(plugin.name, None)
-            if existing_dir is not None:
+            existing_plugin = existing_plugins.get(plugin.name, None)
+            if existing_plugin is not None:
                 if not overwrite:
+                    skipped_plugins.add(plugin.name)
                     logger.info(f"Skipping existing plugin '{plugin.name}'")
                     continue
 
-                logger.debug(f"Overwriting existing plugin '{plugin.name}'")
-                etau.delete_dir(existing_dir)
-                plugin_dir = existing_dir
+                if existing_plugin.builtin:
+                    raise ValueError(
+                        f"Cannot overwrite builtin plugin '{plugin.name}'"
+                    )
+
+                logger.info(f"Overwriting existing plugin '{plugin.name}'")
+                plugin_dir = existing_plugin.directory
+                etau.delete_dir(plugin_dir)
```

Comment:
When does this happen? I would hope that something like `fiftyone plugins download XXX --force` would be required to avoid accidentally deleting something. Possibly exists, just commenting here to make sure we double check this.

Reply:
Correct, user must specify `download_plugin(..., overwrite=True)` or `fiftyone plugins download ... --overwrite` in order to overwrite an existing plugin.

They'll get an error without forcing the overwrite.